IraSoro/peri
# Add French Translation for Peri Application

### Description:
This pull request adds a complete French translation for the Peri app. The translation covers all user interface elements, including the Home Tab, Details Tab, Mark Modal, Welcome Modal, Info Modal, and various phases of the menstrual cycle. 

#### Changes Made:
- Added `fr.ts` file with French translations.
- Updated the `supportedLanguages` array to include French.
- Ensured consistency with existing translations by following the structure and format of the `ru.ts` and `es.ts` files.

#### How to Test:
1. Switch the app language to French in the Preferences menu.
2. Navigate through the app to verify that all text elements are correctly translated.
3. Check for any missing or incorrect translations and report them.

#### Additional Notes:
- This translation was done as part of the [Hacktoberfest](https://hacktoberfest.com/) contribution.
- Please review the translations for accuracy and provide feedback if any changes are needed.

Thank you for considering this contribution!
